### DISCUSSION TITLE
网页跳转优化

### DISCUSSION THREAD

**Discussion by Bill-Haku** [tw93/Pake#143] [Ideas]
对于部分点击跳转会以打开新标签页的形式打开的网站，在使用时会唤起浏览器，体验很不好。希望可以优化为强制在app内打开。

**Comment by tw93** [comment]
噗 其实是前两天有用户提了下 需要默认打开，这一块其实属于萝卜青菜各有所爱，后面我们看看能否直接配置下
你的是哪一个页面
现在应该是非域名内同时a标签是外跳的才会唤起默认浏览器

**Reply by Bill-Haku** [reply]
我使用的情况是bilbili打开动态等页面都会跳转到浏览器。
我认为大多数用户既然选择用打包的app的形式是希望有更加沉浸式的体验，因此最好默认都不要唤起浏览器。

> 现在应该是非域名内同时a标签是外跳的才会唤起默认浏览器

听上去可以，但是我实际使用并不是这样。可能有bug。另外你们也可以考虑在打包的option里面也加一个选项。
